src/repl/completion.rs

```diff
@@ -168,6 +171,33 @@ pub fn path_complete(text: &str) -> Completion {
     }
 }
 
+pub fn write_table(writer: impl Write, words: &[String]) {
+    // TODO: Handle empty case better.
+    let max_length = words.iter().max_by(|a,b| a.len().cmp(&b.len())).unwrap().len() as u16;
+    // TODO: Can this function be called from outside a terminal environment?
+    let (width, _height) = termion::terminal_size().unwrap();
+    let padding = 5;
+    let columns = width / (max_length + padding);
+    let mut tw = TabWriter::new(writer).padding(padding as usize);
```

Comment:
We may want to ditch this albeit neat method of table generation and go for something that will produce better results in our case.

Issues at the moment:
- Sort order is inconsistent with expectation
- Every column is treated as if it was as wide as the widest